recipes/libmatio/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+./autogen.sh
+./configure --quiet --enable-shared --enable-mat73 --enable-extended-sparse --prefix="${PREFIX}" --with-zlib="${PREFIX}" --with-hdf5="${PREFIX}"
+make
+make install
```

Comment:
there is! but it fails on mac os on travis.
the problem is that it needs to be packaged with conda first since conda does some library path mangling and after it gets installed with conda, I need to run `make check`.
Do you know how can I do that?